src/components/table/helpers/mixin-filtering.js

```diff
@@ -20,77 +20,96 @@ export default {
     filterFunction: {
       type: Function,
       default: null
+    },
+    noLocalFiltering: {
+      type: Boolean,
+      default: false
     }
   },
   data() {
     return {
-      // Flag for displaying which empty slot to show, and for some event triggering.
+      // Flag for displaying which empty slot to show and some event triggering
       isFiltered: false
     }
   },
   computed: {
     localFiltering() {
-      return this.hasProvider ? !!this.noProviderFiltering : true
+      return this.hasProvider ? !!this.noProviderFiltering : !this.noLocalFiltering
     },
+    // For watching changes to `filteredItems` vs `localItems`
     filteredCheck() {
-      // For watching changes to filteredItems vs localItems
       return {
         filteredItems: this.filteredItems,
         localItems: this.localItems,
         localFilter: this.localFilter
       }
     },
+    // Sanitized/normalized version of filter prop
     localFilter() {
-      // Returns a sanitized/normalized version of filter prop
+      // Early exit when filtering is disabled
+      if (!this.localFiltering) {
+        return ''
+      }
```

Comment:
Table provider functions still need to be reactive (refreshed) when the filter changes.  Returning an empty string will make it so that the provider is never called when the filter is changed.

Reply:
The early exit always returns `''`, which makes it so that the provider is not triggered (if there is a provider), not the context object changing.

Reply:
Damn, forgot to save the file again...